### PR TITLE
Mail room ver 2. - Missing feature added

### PR DESCRIPTION
**Pull request by yilmazkennan**
Missing feature was added to the code.  Resubmitting the assignment

### CODE REVIEW

File: students/KennanY/Lesson3/mailroom.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+donations = [
+            ['Bill Gates', 165234, 42335.37, 21000,4400],
+            ['Jack Ma', 94834.44, 239102.23,9984],
+            ['Jeff Bezos', 1234534,234123],
+            ['Kate Jones', 948374.44, 987362.12],
+            ['Walt Kistler', 1234, 4444, 38476.34],
+            ['Jerry Paros', 110,44,85,8854]]
+
+def createReport():
```
Comment (natasha-aleksandrova) [inline students/KennanY/Lesson3/mailroom.py]: functions (and var names) should be snake_case -> create_report

File: students/KennanY/Lesson3/mailroom.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        response=input("Please select and option (1,2 or 3)")
+        if str(response)=='1':
+            send_ThankYou()
+        elif str(response)=='2':
+            createReport()
+        elif str(response)=='3':
+            print('Finished ...')
+            break
+        else:
+            select=True
```
Comment (natasha-aleksandrova) [inline students/KennanY/Lesson3/mailroom.py]: you don't need the `select` var, it is never set to `False`.. use `while True` and your `break` statement will end the loop

**Comment by natasha-aleksandrova** [comment]
@yilmazkennan adjusted your grade, thank you!

**Comment by yilmazkennan** [comment]
Thank you for the feedback.  I am using PyCharm as my IDE.

**Comment by natasha-aleksandrova** [comment]
@yilmazkennan if you're using Pycharm you should see a lot of underline type of warnings, for example:
screen shot 2018-12-10 at 8 06 30 am


those are the violations

**Comment by yilmazkennan** [comment]
I dont see the errors but I used the tab key while using Charm.  I can see the vertical tab lines

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request merged**